scripts/build-images

```diff
@@ -12,22 +12,22 @@ awk '{print $1}' << EOF > build/images-core.txt
 EOF
 
 xargs -n1 -t docker image pull --quiet << EOF >> build/images-core.txt
-    ${REGISTRY}/rancher/hardened-kubernetes:${KUBERNETES_IMAGE_TAG}
-    ${REGISTRY}/rancher/hardened-coredns:v1.9.3-build20220613
-    ${REGISTRY}/rancher/hardened-cluster-autoscaler:v1.8.5-build20211119
-    ${REGISTRY}/rancher/hardened-dns-node-cache:1.21.2-build20211119
-    ${REGISTRY}/rancher/hardened-etcd:${ETCD_VERSION}-build20220504
-    ${REGISTRY}/rancher/hardened-k8s-metrics-server:v0.6.1-build20220929
+    ${REGISTRY}/bcibase/hardened-kubernetes:${KUBERNETES_IMAGE_TAG}
+    ${REGISTRY}/bcibase/hardened-coredns:v1.9.3-build20220613
+    ${REGISTRY}/bcibase/hardened-cluster-autoscaler:v1.8.5-build20211119
+    ${REGISTRY}/bcibase/hardened-dns-node-cache:1.21.2-build20211119
+    ${REGISTRY}/bcibase/hardened-etcd:${ETCD_VERSION}-build20221004
+    ${REGISTRY}/bcibase/hardened-k8s-metrics-server:v0.6.1-build20220929
     ${REGISTRY}/rancher/klipper-helm:v0.7.3-build20220613
     ${REGISTRY}/rancher/pause:${PAUSE_VERSION}
     ${REGISTRY}/rancher/mirrored-ingress-nginx-kube-webhook-certgen:v1.1.1
-    ${REGISTRY}/rancher/nginx-ingress-controller:nginx-1.2.1-hardened7
-    ${REGISTRY}/rancher/rke2-cloud-provider:${CCM_VERSION}
+    ${REGISTRY}/bcibase/nginx-ingress-controller:nginx-1.2.1-hardened7
+    ${REGISTRY}/bcibase/rke2-cloud-provider:v1.25.2-build20221003
```

Comment:
I think we still use `${CCM_VERSION}` in other places - are we attempting to drop it?
* https://github.com/rancher/rke2/blob/master/scripts/version.sh#L38
* https://github.com/rancher/rke2/blob/master/scripts/build-binary#L31